<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShopWindow</class>
 <widget class="QMainWindow" name="ShopWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="shopkeeper">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/resources/images/shopkeeper.png);
background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dialog">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>1200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border-style: solid;
border-width: 2px;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="select">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>230</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="board">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>511</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/resources/images/shopboard.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="shoplist">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>321</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>350</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QStackedWidget" name="next_floor_page">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="next_floor">
      <property name="geometry">
       <rect>
        <x>1040</x>
        <y>600</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
   <widget class="QProgressBar" name="PlayerMPBar">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>350</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: gold;</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="PlayerATK_title">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>410</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/resources/images/ATK.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="PlayerEXP">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>260</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/resources/images/exp.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="PlayerAP_title">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>380</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/resources/images/AP.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="PlayerHP">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>320</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/resources/images/HP.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="PlayerMP">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>350</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/resources/images/MP.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="PlayerEXPBar">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>260</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: gold;</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="PlayerAPBar">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>380</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: gold;</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="PlayerFunds">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>290</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: gold;</string>
    </property>
    <property name="text">
     <string>Funds</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="PlayerName_title">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>200</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/resources/images/Name.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="PlayerLevel">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>230</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: gold;</string>
    </property>
    <property name="text">
     <string>Level</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="PlayerLevel_title">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>230</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/resources/images/Level.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="PlayerDEF_title">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>440</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/resources/images/DEF.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="PlayerDEF">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>440</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: gold;</string>
    </property>
    <property name="text">
     <string>DEF</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="PlayerName">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>200</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: gold;</string>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="PlayerHPBar">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>320</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: gold;</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="PlayerATK">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>410</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: gold;</string>
    </property>
    <property name="text">
     <string>ATK</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="PlayerFunds_title">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>290</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/resources/images/funds.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>shopkeeper</zorder>
   <zorder>dialog</zorder>
   <zorder>board</zorder>
   <zorder>shoplist</zorder>
   <zorder>select</zorder>
   <zorder>back</zorder>
   <zorder>PlayerMPBar</zorder>
   <zorder>PlayerATK_title</zorder>
   <zorder>PlayerEXP</zorder>
   <zorder>PlayerAP_title</zorder>
   <zorder>PlayerHP</zorder>
   <zorder>PlayerMP</zorder>
   <zorder>PlayerEXPBar</zorder>
   <zorder>PlayerAPBar</zorder>
   <zorder>PlayerFunds</zorder>
   <zorder>PlayerName_title</zorder>
   <zorder>PlayerLevel</zorder>
   <zorder>PlayerLevel_title</zorder>
   <zorder>PlayerDEF_title</zorder>
   <zorder>PlayerDEF</zorder>
   <zorder>PlayerName</zorder>
   <zorder>PlayerHPBar</zorder>
   <zorder>PlayerATK</zorder>
   <zorder>PlayerFunds_title</zorder>
   <zorder>next_floor_page</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
